### PR TITLE
Emily arm tuning

### PR DESCRIPTION
**Pull request by emilycxliu**
## Describe your changes

- [ ] Bug fix
- [ ] New feature

Explain what you did:
> ...

## Issue ticket (add number)

> https://github.com/FRC2706/2024-2706-Robot-Code/issues/...

## Checklist before requesting a review
- [ ] I have performed a self-review of my code.
- [ ] I have tested thoroughly.
- [ ] I have written documentation and explained about my changes to the best of my ability.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 1]
```diff
@@ -54,7 +55,7 @@ public class ArmSubsystem extends SubsystemBase{
     //spark absolute encoder
     private SparkAbsoluteEncoder m_absEncoder;  
     //embedded relative encoder
-    //private RelativeEncoder m_Encoder;
+    private RelativeEncoder m_encoder;
```
Comment (ErikCald) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java]: Remove this encoder. The AbsoluteEncoder is now wired directly into the sparkmax which replaces the RelativeEncoder entirely.

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 2]
```diff
@@ -150,12 +151,24 @@ public void burnFlash() {
...
     public void isAtSetpoint() {
-      
+    }
+
+    public void setArmIdleMode(IdleMode mode) {
+      m_arm.setIdleMode(mode);
+    }
+
+    public void testFeedForward(double additionalVoltage) {
+      double voltage = additionalVoltage + calculateFF(m_encoder.getPosition());
```
Comment (ErikCald) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java]: Change `m_encoder.getPosition()` to `m_absEncoder.getPosition()`.

**Review by ErikCald — COMMENTED** [review]
I left 2 comments that need to be addressed before merging but I have not set it as Request changes so that you can address it and merge it today.

Don't forget to bind the test FF command to a button.

Other than the 2 comments I would approve.

**Review by weizhang-2706 — APPROVED** [review]

**Review by ErikCald — APPROVED** [review]
Make sure to test on the robot before merging

**Pull request merged**